docs/ldml/tr35.md

```diff
@@ -1030,7 +1030,7 @@ The BCP 47 form for keys and types is the canonical form, and recommended. Other
 <tr><td>"cu"<br>(currency)</td>
     <td>Currency type</td>
     <td><i>ISO 4217 code,</i><p><i>plus others in common use</i></p></td>
-    <td><p>Codes consisting of 3 ASCII letters that are or have been valid in ISO 4217, plus certain additional codes that are or have been in common use. The list of countries and time periods associated with each currency value is available in <a href="tr35-numbers.md#Supplemental_Currency_Data">Supplemental Currency Data</a>, plus the default number of decimals.</p><p>The XXX code is given a broader interpretation as <i>Unknown or Invalid Currency</i>.</p></td></tr>
+    <td><p>Well-formed codes are of the form <code>[A-Za-z]{3}</code>, with the canonical format being <code>[A-Z]{3}</code>. The valid codes are or have been valid in ISO 4217, plus certain additional codes that are or have been in common use. The list of countries and time periods associated with each currency value is available in <a href="tr35-numbers.md#Supplemental_Currency_Data">Supplemental Currency Data</a>, plus the default number of decimals.</p><p>The XXX code is given a broader interpretation than in ISO 4217, as <i>Unknown or Invalid Currency</i>.</p></td></tr>
```

Comment:
Line breaks make this easier to comment on.

The valid code are or have been => "The valid codes are ones that are or have been"

The "plus the default number of decimals" feels oddly tacked on. I would reverse this sentence to be something like:

> [Supplemental Currency Data](tr35-numbers.md#Supplemental_Currency_Data) provides the list of 
> countries/regions and time periods associated with each currency code.
> It also provides the default number of decimal places.

Note "currency/region", to avoid sensitivity over calling e.g. `HK` (or similar) a country (not to mention some currencies are region codes, such as `EUR` or some of the African basket ones)

Reply:
Fixed these, thanks.

As to linebreaks. One of our tasks is to finish the conversion to markdown, which is mostly fixing the tables that didn't translate, and then move towards Semantic linebreaks.